grpc/src/main/java/com/linecorp/armeria/server/grpc/HttpJsonTranscodingService.java

```diff
@@ -651,7 +651,7 @@ private static JsonNode getBodyContent(AggregatedHttpRequest request) {
         try {
             return mapper.readTree(request.contentUtf8());
         } catch (JsonProcessingException e) {
-            return null;
+            throw new IllegalArgumentException("error ocurred while parsing json request", e);
```

Comment:
`ocurred` -> `occurred` Or, just
` throw new IllegalArgumentException("invalid JSON request", e);`

Reply:
Question: What's the expected behavior when the content type is missing? Shouldn't we also raise an exception?